### PR TITLE
Adds reload_enabled_scripts shortcut

### PR DESCRIPTION
**Pull request by jeremypepper**
Shortcut is by default same as playlunky (ctrl +f5)

### CODE REVIEW

**Review by Malacath-92 — CHANGES_REQUESTED** [review]

File: src/injected/ui.cpp [hunk 1]
```diff
@@ -120,6 +120,7 @@ std::map<std::string, int> keys{
     {"mouse_clone", 0x505},
     {"mouse_destroy", 0x405},
     {"mouse_destroy_unsafe", 0x605},
+    {"reload_enabled_scripts", 0x173} // ctrl + f4 same as playlunky
```
Comment (gmjosack) [inline src/injected/ui.cpp:123]: Doesn't playlunky use ctrl + f5 to reload? I though ctrl + f4 only showed the menu

https://github.com/spelunky-fyi/Playlunky/blob/d98f09b1bdd680437248913524dd74d7dbc1c7d2/source/playlunky/mod/mod_manager.cpp#L574

Comment (Malacath-92) [reply src/injected/ui.cpp:123]: Yes, you are right. That is my mistake, I believe I said that on Discord.

File: src/injected/ui.cpp [hunk 2]
```diff
@@ -3946,3 +3951,14 @@ void init_ui()
...
+void reload_enabled_scripts()
+{
+    for (auto& [script_name, script] : g_scripts)
+    {
+        if (script->is_enabled())
+        {
+            load_script(script->get_file(), true);
+        }
+    }
+}
```
[the remark this answers is not in the document]
Comment (jeremypepper) [reply src/injected/ui.cpp]: done. can we configure clang-format to do this?

File: src/injected/ui.cpp (hunk 1 above)
Comment (jeremypepper) [reply src/injected/ui.cpp]: fixed

**Review by Malacath-92 — APPROVED** [review]

File: src/injected/ui.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (Malacath-92) [reply src/injected/ui.cpp]: A bit late response, but clang-format does not have an option for new-lines at eof. Idk why, but it's a contentious topic.

**Pull request merged**